### PR TITLE
Fixed typos, broken links, tidied up language

### PR DESCRIPTION
**Pull request by LindsayHill**
- Fixed a few small typos
- Tidied up some language
- Removed AIO references from Installing Enterprise section

### CODE REVIEW

**Comment by emedvedev** [comment]
Grammar time, yay! This PR makes me happy.

**Comment by emedvedev** [comment]
@LindsayHill: LGTM, merging. Don't forget to create a PR against the 1.4 branch as well to update current docs.

File: docs/source/install/deb.rst [hunk 1]
```diff
@@ -12,12 +12,12 @@ Supported versions
 ------------------
 
 We support Ubuntu 14.04, and test on `Ubuntu Server 14.04 LTS (HVM) Amazon AWS AMI <https://aws.amazon.com/marketplace/pp/B00JV9TBA6/ref=srh_res_product_title?ie=UTF8&sr=0-3&qid=1457037882965>`_
-and `puppetlabs/ubuntu-14.04-64-nocm Vagrant box <https://atlas.hashicorp.com/puppetlabs/boxes/ubuntu-14.04-64-nocm>`_. Other Debian based distributions and versions will likely work with some tweaks, you are welcome to try and report success to the `community <https://stackstorm.com/community-signup>`_.
+and `puppetlabs/ubuntu-14.04-64-nocm Vagrant box <https://atlas.hashicorp.com/puppetlabs/boxes/ubuntu-14.04-64-nocm>`_. Other Debian based distributions and versions will likely work with some tweaks. You are welcome to try - please report success to the `community <https://stackstorm.com/community-signup>`_.
 
 Sizing the server
 -----------------
-While the system can operate with less equipped servers, these are recommended
```
Comment (Kami) [inline docs/source/install/deb.rst]: "less equipped servers" - lol, that was an "interesting" phrase :P

**Comment by Kami** [comment]
Nice, thanks :+1: 

(so easy to spot which docs were written by Dmitri and myself :P)

**Comment by LindsayHill** [comment]
Thanks - will also create PR against 1.4. Will be a few more tidy-up changes coming over the next week.

**Pull request merged**